package.json

```diff
@@ -6,15 +6,15 @@
   "scripts": {
     "lint": "eslint",
     "make-model": "npx sequelize-cli model:generate --name",
-    "migrate": "npx sequelize-cli db:migrate",
-    "seed": "npx sequelize-cli db:seed:all",
+    "migrate": "NODE_ENV=$NODE_ENV npx sequelize-cli db:migrate --config=./src/database/config/config.js",
+    "seed": "NODE_ENV=$NODE_ENV npx sequelize-cli db:seed:all --config=./src/database/config/config.js",
     "seed-down": "npx sequelize-cli db:seed:undo:all",
     "make-seed": "npx sequelize-cli seed:generate --name ",
-    "down": "npx sequelize-cli db:migrate:undo",
+    "down": "NODE_ENV=$NODE_ENV npx sequelize-cli db:migrate:undo  --config=./src/database/config/config.js",
     "downall": "npx sequelize-cli db:migrate:undo:all",
     "dev": " cross-env NODE_ENV=development nodemon -r @babel/register src/index",
     "start": "cross-env NODE_ENV=production node -r @babel/register src/index",
-    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/tests/ --recursive --exit --timeout 120000",
+    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/tests/ tests.js/ --recursive --exit --timeout 120000",
```

Comment:
remove `tests.js/`

Reply:
There was a time @dushimeemma  told us to use the 'test.js' folder for our tests. that's where i'm putting my test and that's why i have that on the test command.